### PR TITLE
Format and path options

### PR DESCRIPTION
**Pull request by 7sedam7**
Especially when using this on a big trace path, the png image is unusable and svg fits better.
To not limit to any of the choices I'm proposing to have an option to set it, but still, leave the default as you made it.

### CODE REVIEW

**Comment by 7sedam7** [comment]
@matugm Can you please take a look and merge if it's ok :)

File: lib/visual_call_graph/graph_manager.rb [hunk 1]
```diff
@@ -32,8 +32,10 @@ def get_node_name(event)
     end
   end
 
-  def output(*args)
-    @g.output(*args)
+  def output
+    format = @options[:format] || :png
+    path   = @options[:path] || "#{Dir.pwd}/call_graph.#{format.to_s}"
+    @g.output(format.to_sym => path)
```
Comment (matugm) [inline lib/visual_call_graph/graph_manager.rb:39]: Can you add a blank line between the local variables & the method call? I think it makes the code easier to read :)

**Review by matugm — CHANGES_REQUESTED** [review]

**Comment by 7sedam7** [comment]
good suggestion @matugm and done :)

**Review by matugm — APPROVED** [review]

**Comment by 7sedam7** [comment]
Thanks for the approve, can you merge please :)

**Pull request merged**